Flutter/datagrid/editing.md

```diff
@@ -564,6 +564,90 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
+## How to check whether current cell is in editing mode
+
+You can check whether current cell is in editing or not by using `DataGridController.isCurrentCellInEditing` property.
```

Comment:
You can check whether the current cell is in editing or not by using the `DataGridController.isCurrentCellInEditing` property.